docs/docs/rest-api.md

```diff
@@ -195,6 +198,30 @@ Valid constraint operators are one of ["UNIQUE", "CLUSTER",
 "GROUP_BY"]. For additional information on using placement constraints see
 the [Constraints doc page]({{ site.baseurl }}/docs/constraints.html).
 
+##### acceptedResourceRoles
+
+Optional. A list of resource roles. Marathon considers only resource offers with roles in this list for launching
+tasks of this app. If you do not specify this, Marathon considers all resource offers except you override the
```

Comment:
If you do not specify this, Marathon considers all resource offers that have been configured by the `--default_accepted_resource_roles` command line flag. Note that this defaults to "*" by default.